sevenhills.html

```diff
@@ -0,0 +1,290 @@
+                            <g id="About-Crossroads" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"
+                                fill-opacity="0.35">
+                                <g id="About-Us---Final" transform="translate(-84.000000, -1891.000000)" fill="#E8E2D0"
+                                    fill-rule="nonzero">
+                                    <polygon id="ornaments-arrow-down"
+                                        points="92 1891 92 1915 98 1915 91 1929 84 1915 90 1915 90 1891"></polygon>
+                                </g>
+                            </g>
+                        </svg>
+                    </div>
+                    <p class="font-family-base-bold text-uppercase text-orange flush">the hill of:</p>
+                    <h2 class="section-header text-white flush">reproduction</h2>
+                        <p class="push-top">Reproduction is about recognizing something that God has created and then reproducing it in
+                            his name. We want to send God’s message out into the world by reproducing everything from
+                            the weekend program to CDs of each message series. Crossroads wants to be a place where
+                            Christ-followers reproduce Christ-followers, leaders reproduce leaders and churches
+                            reproduce other churches. (From Matthew 28:19-20 in the Bible)</p>
+    
+                </div>
+            </div>
+        </div>
+    </div>
+</section>
+
+<!-- Growth -->
+<section class="soft-bottom bg-blue">
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-md-7 text-center soft-ends push-top">
+                <img class="seven-hills-img-size"
+                    src="//crds-media.imgix.net//1DT6n7TlYDWF4v4lMH43Od/bd60a87b9d94459cd0908eb82c32c858/nothing-changed.png">
+            </div>
+            <div class="col-md-5 content-center content-right">
+                <div class="portrait-card-content">
+                    <div class="portrait-card-arrow">
+                        <svg width="14px" height="38px" viewBox="0 0 14 38" version="1.1"
+                            xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+                            <g id="About-Crossroads" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd"
+                                fill-opacity="0.35">
+                                <g id="About-Us---Final" transform="translate(-84.000000, -1891.000000)" fill="#E8E2D0"
+                                    fill-rule="nonzero">
+                                    <polygon id="ornaments-arrow-down"
+                                        points="92 1891 92 1915 98 1915 91 1929 84 1915 90 1915 90 1891"></polygon>
+                                </g>
+                            </g>
+                        </svg>
```

Comment:
Considering this SVG is repeated a bunch of times in this file, you might consider sticking this into a partial and/or moving the SVG into a standalone file and using the `<img />` to render it here.